command/agent.go

```diff
@@ -1047,3 +1050,19 @@ func (c *AgentCommand) handleMetrics() http.Handler {
 		}
 	})
 }
+
+func (c *AgentCommand) handleQuit() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Only handle POST/PUT requests
+		switch r.Method {
+		case http.MethodPost:
```

Comment:
Out of curiosity what is the reason for supporting multiple methods?

Reply:
I didn't give it much thought tbh, I just didn't change it. Just POST is better though - changed.